src/EditorFeatures/CSharpTest/EditAndContinue/ActiveStatementTests.cs

```diff
@@ -1804,7 +1804,7 @@ class C
             var active = GetActiveStatements(src1, src2);
 
             edits.VerifyRudeDiagnostics(active,
-                Diagnostic(RudeEditKind.ModifiersUpdate, "const int a = 1", "const field"));
```

Comment:
These contain a lot of language keywords that should not be localized (especially in the Rude Edit messages).  How are we going to annotate these so that they don't get translated?

For example, if I tried to type the following code in EnC:
```
Dim r = Aggregate x In {1, 2, 3} Into All(x < 4)
```
The rude edit message should contain the word "Aggregate".  It would be completely wrong if, for instance, in German, this was translated to "Aggregat".

If no such mechanism exists, then in my opinion, these should be hard-coded const strings (in the product code and in tests).